index.js

```diff
@@ -55,6 +57,7 @@ async function run(rootDir, options = {}) {
     log(' 👏  No unused translations were found!');
   } else {
     log(` ⚠️   Found ${chalk.bold.yellow(unusedTranslations.size)} unused translations!`);
+    log(' use --fix to remove all unused translations');
```

Comment:
```suggestion
    log('     You can use --fix to remove all unused translations.');
```

so that this aligns a little better :)

Reply:
and I guess we should only log this line if `shouldFix` is not true?